docs/source/release/v6.3.0/indirect_geometry.rst

```diff
@@ -15,6 +15,7 @@ New Features
 - In Indirect Data Analysis fitting tabs a button has been added that will unify the fit range for all spectra selected.
 - The Bayes Fortran libraries can now be installed on OSX and Linux using the bayesfitting python package https://pypi.org/project/bayesfitting/0.1.0/. This can be installed with `python -m pip install bayesfitting`
   using the Mantid Python executable.
+- VesuvioAnalysis now allows defining constraints for more than two masses.
```

Comment:
Could we add something like 
```
using the ConstraintsProfile, see :ref:`VesuvioAnalysis <algm-VesuvioAnalysis>`
```

so users can easily jump to the docs / see the parameter we've updated?

Reply:
That is a good idea. I've added it to the release note.